.gitignore

```diff
@@ -1 +0,0 @@
-node_modules/
```

Comment:
I'll leave it, then. I'd point out that global `.gitignore` is very handy, however.  Especially now with `npm@5` and `package-lock.json`.